src/shared/stores/streamEditor.tsx

```diff
@@ -268,7 +264,7 @@ export const useStreamEditorStore = create<Actions & State>((set, get) => {
                     next.loadError = null
                 })
             } catch (e: unknown) {
-                if (isMessagedObject(e) && /not_found/i.test(e.message)) {
+                if (isMessagedObject(e) && /not.found/i.test(e.message)) {
```

Comment:
This solves an issue where streams that could not be found produced a "general" error when they should've produced a 404 error (current original error message is "Steam not found" and not "stream_not_found something something").